template/client/store/actions.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import * as mutationTypes from '~/store/types'
```

Comment:
Could you please resort imports as follows:

```js
import comments from '~/logics/api/comments'
import * as mutationTypes from '~/store/types'

import type { ActionContext } from 'vuex'
import type { StateType } from '~/types'
```

This is a linting issue: https://github.com/wemake-services/wemake-vue-template/issues/495
My bad, that I have not included it yet. Sorry!